widgetry/src/shaders/vertex_140.glsl

```diff
@@ -22,10 +22,18 @@ void main() {
     float screen_x = (position[0] * zoom) - transform[0];
     float screen_y = (position[1] * zoom) - transform[1];
 
-    // Translate that to normalized device coordinates (NDC)
+    // Translate position to normalized device coordinates (NDC)
     float x = (screen_x / window[0] * 2.0) - 1.0;
     float y = (screen_y / window[1] * 2.0) - 1.0;
-    float z = position[2] + window[2];
+
+    // largest absolute value for layer base z values in drawing.rs (i.e. MAPSPACE_Z vs. TOOLTIP_Z)
+    float z_range = 2.0;
```

Comment:
This solution is kind of brittle in that it requires the programmer to keep in sync with any new layers that we add to drawing.rs, but since it never changes once the app is compiled, it seemed not worth it to plumb through the value in code.

Reply:
This coupling is localized and totally fine, and spending a uniform value on it is silly, so LGTM. Thanks for fixing!